doc/specifications/proposed/translations/removal.md

```diff
@@ -0,0 +1,55 @@
+# Removal of Content Object Translations
+
+## Deleting single translation from a content object
+
+Sometimes there is a need to delete single translation of a content object without deleting
+the object itself.
+
+To aid that, PHP API introduces the `deleteTranslation` method on `ContentService` specified as:
+
+```php
+/**
+ * Delete content object translation by creating new version draft without the specified translation.
+ *
+ * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException if the specified translation
+ *         is the only one content object has.
+ *
+ * @param \eZ\Publish\API\Repository\Values\Content\VersionInfo $versionInfo
+ * @param string $languageCode
+ *
+ * @return \eZ\Publish\API\Repository\Values\Content\Content Content draft
+ */
+public function deleteTranslation(VersionInfo $versionInfo, $languageCode);
```

Comment:
naming.. this method actually does not delete anything, so maybe it's more of a
`createContentDraftWithouthTranslation()` But yeah, long, and it would suggests it should take contentInfo as argument which we don't really care about here.

Suggestions @Nattfarinn ? :)